### ISSUE TITLE
[Bug]: Voiceover is announcing two unnecessary roles for "Search" box in "Tag Picker component".

### ISSUE THREAD

**Issue by rupeshsah1** [microsoft/fluentui#28220]
### Library

React V8

### System Info

```shell
https://developer.microsoft.com/en-us/fluentui#/controls/web/pickers
```

### Are you reporting Accessibility issue?

Yes

### Reproduction

https://codepen.io/RupeshReshav/pen/VwVemzz

### Bug Description

Actual:
Voice over is announcing two unnecessary incorrect roles such as "List items popup and menu item popup" for "Search" edit field in "TagPicker Component".

Expected:
Voiceover should announce correct role edit field popup for "Search" edit field in "TagPicker Component".

Explanation:
The input box has both role and aria-haspopup defined. So, it says both the screen reader attribute.

Comboboxes have an implicit aria-haspopup value of listbox, so including this attribute is optional.

```
<input autocapitalize="off" autocomplete="off" 
aria-autocomplete="both" spellcheck="false"
 placeholder="Search" class="ms-BasePicker-input input-619" 
id="combobox-id__242" aria-expanded="false" data-lpignore="true" 
value="" style="font-family: inherit;" 
**aria-haspopup="listbox" 
role="combobox">**
```

### Requested priority

Normal

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.
